.changeset/gold-frogs-yell.md

```diff
@@ -0,0 +1,16 @@
+---
+'@directus/sdk': major
+---
+
+Added support for LDAP login and enhanced parameter consistency for refresh and logout commands
+
+::: notice
+The `login` method now accepts a payload object instead of separate email and password parameters. 
+This enables support for both standard and LDAP authentication, but requires updates to code that previously called `sdk.login(email, password)`. 
+The new usage is `sdk.login({ email, password })` or `sdk.login({ identifier, password })` for LDAP.
+
+The `refresh` and `logout` method parameters have been updated to be consistent across usage. The functions now accept an options object instead of their previous regular parameters.
```

Comment:
```suggestion
The `refresh` and `logout` command method parameters have been updated to be consistent across usage. The functions now accept an options object instead of their previous regular parameters.
```